spec/rails_helper.rb

```diff
@@ -7,6 +7,7 @@
 # Prevent database truncation if the environment is production
 abort('The Rails environment is running in production mode!') if Rails.env.production?
 require 'rspec/rails'
+require 'support/active_resource_mock_helper'
```

Comment:
Might be worth while adding

```ruby
# Add additional requires below this line. Rails is not loaded until this point!
Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```

instead so all new files in the support folder will be included